### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by brassy-**

### CODE REVIEW

**Comment by bwinton** [comment]
I'm not a content reviewer.  I'm just here for the code…  😉
@aminalhazwani, @pwalm, or @designakt would be better choices, I think.

File: icons/new/fullscreen-exit-16.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+  <title>fullscreen-exit-16</title>
```
Comment (aminalhazwani) [inline icons/new/fullscreen-exit-16.svg]: I think we can remove the title

File: icons/new/screenshot-16.svg [hunk 2]
```diff
@@ -2,5 +2,9 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
-  <path fill="context-fill" d="M4 0a2 2 0 0 0-2 2h2zm6 0H8v2h2zm4 0v2h2a2 2 0 0 0-2-2zM7 0H5v2h2zm6 0h-2v2h2zm1 5h2V3h-2zm0 6a2 2 0 0 0 2-2h-2zm0-3h2V6h-2zm-.441-2.167a2 2 0 0 0-2.785-.492L6.939 8.172l-1.968-1.38A2.457 2.457 0 0 0 5 6.5a2.488 2.488 0 0 0-1-1.989V3H2v1.05a2.5 2.5 0 1 0 1.958 4.474l1.242.869-1.4.982A2.494 2.494 0 1 0 5 12.5a2.508 2.508 0 0 0-.05-.492zM2.5 7.75A1.25 1.25 0 1 1 3.75 6.5 1.25 1.25 0 0 1 2.5 7.75zm0 6a1.25 1.25 0 1 1 1.25-1.25 1.25 1.25 0 0 1-1.25 1.25zm5.4-2.429l3.017 2.3a2 2 0 0 0 2.785-.492L9.64 10.1z"/>
+  <title>screenshot-16</title>
```
Comment (aminalhazwani) [inline icons/new/screenshot-16.svg]: I think we can remove the title

File: icons/addons/plugins.svg [hunk 3]
```diff
@@ -1,4 +1,4 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. --> 
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M16,13V9a3,3,0,0,0-3-3h0V5a1,1,0,0,0-1-1H9.8a.8.8,0,0,0-.8.8V6H7V4.8A.8.8,0,0,0,6.2,4H3.8a.8.8,0,0,0-.8.8V5.2a.8.8,0,0,1-.8.8h0A2.2,2.2,0,0,0,0,8.2V13.8A2.2,2.2,0,0,0,2.2,16H13A3,3,0,0,0,16,13ZM2.8,14a.8.8,0,0,1-.8-.8V8H14v5.2a.8.8,0,0,1-.8.8Z" fill="context-fill" /></svg>
\ No newline at end of file
+<svg version="1.1" id="plugins-icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve"> <path id="plugins" d="M0,13c0,1.7,1.3,3,3,3h10.8c1.2,0,2.2-1,2.2-2.2V8.2C16,7,15,6,13.8,6l0,0l0,0C13.4,6,13,5.6,13,5.2V4.8l0,0 C13,4.4,12.6,4,12.2,4H9.8c0,0,0,0,0,0C9.4,4,9,4.4,9,4.8V6H7V4.8C7,4.4,6.6,4,6.2,4c0,0,0,0,0,0H3.8C3.4,4,3,4.4,3,4.8l0,0v0.4 C3,5.6,2.6,6,2.2,6l0,0l0,0C1,6,0,7,0,8.2L0,13z M2.8,14L2.8,14C2.4,14,2,13.6,2,13.2V8h12v5.2l0,0c0,0.4-0.4,0.8-0.8,0.8H2.8z"/></svg>
```
Comment (aminalhazwani) [inline icons/addons/plugins.svg]: There's a space between `xml:space="preserve">` and `<path`

File: icons/addons/services.svg [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. --> 
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M0,4V8.7a10.45,10.45,0,0,0,0,1.84A3.1,3.1,0,0,0,3,13c1.28.17.86,1,1,2,.18,1.26,1.24,1.09,2,.51l2.34-1.75A2.65,2.65,0,0,1,9.77,13c1.76-.15,3.95.52,5.35-.88S16,8.63,16,7s.39-3.86-.88-5.12S12.36,1,10.92,1H4.38C3.92,1,3.46,1,3,1A3,3,0,0,0,0,4Zm14,6.2c0,1.38-3.81.78-4.67.8C8.05,11,7,12.26,6,13c0-.47.14-1.27-.16-1.67S4.7,11,4.2,11c-1,0-2.15.22-2.2-1.08S2,7.44,2,6.21A8.67,8.67,0,0,1,2,3.55C2.33,2.77,3.56,3,4.21,3h7.72c.64,0,1.74-.21,2,.55a3.47,3.47,0,0,1,0,1.11V10.2Z" fill="context-fill" /></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M4,15V14a1,1,0,0,0-1-1H3a3,3,0,0,1-3-3V4A3,3,0,0,1,3,1H13a3,3,0,0,1,3,3v6a3,3,0,0,1-3,3H10.34a3,3,0,0,0-1.8.6l-2.93,2.2A1,1,0,0,1,4,15Zm1-4a1,1,0,0,1,1,1v1l2.4-1.8A1,1,0,0,1,9,11h4a1,1,0,0,0,1-1V4a1,1,0,0,0-1-1H3A1,1,0,0,0,2,4v6a1,1,0,0,0,1,1Z" fill="context-fill"/></svg>
```
Comment (aminalhazwani) [inline icons/addons/services.svg]: Sometimes we indent and sometimes we do not. I would stick to only one convention :)

Comment (brassy-) [inline icons/addons/services.svg]: Let's have everything minified. I just pushed all the icons minified

**Comment by aminalhazwani** [comment]
@bwinton we can merge!

**Pull request merged**